sample-code/examples/java/junit/src/test/java/com/saucelabs/appium/page_object/android/ApiDemosListViewScreenChaided.java

```diff
@@ -33,48 +29,48 @@
 	 * It was done so just for obviousness
 	 */
 
-	@AndroidFindBys({
+	@AndroidFindBySet({
 		@AndroidFindBy(uiAutomator = "new UiSelector().resourceId(\"android:id/list\")"), //the searching

```

Comment:
Yes that is for allpossible but for chaining we dont need set so we can have normal repetitive @AndroidFindBy or with @HowToUseLocator=CHAIN